runtime/vm/jvminit.c

```diff
@@ -3564,6 +3567,14 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 			vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_CRIU_SUPPORT;
 		}
 	}
+
+	{
+		IDATA enableCRIUNonPortableMode = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXENABLECRIUNONPORTABLEMODE, NULL);
+		IDATA disableCRIUNonPortableMode = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXDISABLECRIUNONPORTABLEMODE, NULL);
+		if (enableCRIUNonPortableMode > disableCRIUNonPortableMode) {
+			vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_CRIU_NON_PORTABLE_MODE;
```

Comment:
This can allocate the `vm->checkpointState` and set a bit there rather than using a `extendedRuntimeFlags2` bit